src/coreComponents/physicsSolvers/fluidFlow/FlowSolverBase.cpp

```diff
@@ -109,29 +109,7 @@ void FlowSolverBase::RegisterDataOnMesh( Group * const MeshBodies )
       subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::aperture0String )->
         setDefaultValue( faceRegion.getDefaultAperture() );
       subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::effectiveApertureString )->
-        setApplyDefaultValue( subRegion.getWrapper< array1d< real64 > >( FaceElementSubRegion::
-                                                                           viewKeyStruct::
-                                                                           elementApertureString )->getDefaultValue() )->
-        setPlotLevel( PlotLevel::LEVEL_0 );
-    } );
-
-    //TODO: I think these 2 subregions should be treated in the same way. Need to unify the region
-    elemManager->forElementSubRegionsComplete< EmbeddedSurfaceSubRegion >( [&]( localIndex const,
-                                                                                localIndex const,
-                                                                                ElementRegionBase & region,
-                                                                                EmbeddedSurfaceSubRegion & subRegion )
-    {
-      EmbeddedSurfaceRegion & embSurfRegion = dynamicCast< EmbeddedSurfaceRegion & >( region );
-
-      subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::referencePorosityString )->
-        setApplyDefaultValue( 1.0 );
-
-      subRegion.registerWrapper< array1d< R1Tensor > >( viewKeyStruct::permeabilityString )->setPlotLevel( PlotLevel::LEVEL_0 );
-      subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::gravityCoefString )->setApplyDefaultValue( 0.0 );
-      subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::aperture0String )->
-        setDefaultValue( embSurfRegion.getDefaultAperture() );
-      subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::effectiveApertureString )->
-        setApplyDefaultValue( subRegion.getWrapper< array1d< real64 > >( EmbeddedSurfaceSubRegion::
+        setApplyDefaultValue( subRegion.getWrapper< array1d< real64 > >( SurfaceElementSubRegion::
```

Comment:
The effective aperture should go through a calculation rather than be user input. It isn't clear that this is correct here.

Reply:
I agree with you about the effective aperture but in case one wants to do a flow simulation without any mechanics there is still the need to define an effective aperture in the fractures and I thought it could be a user input. I think the effective aperture is only updated in the HydrofractureSolver and this why it needs to be initialised here. This way there is no need to go through the HydrofracureSolver nor to have a mechanicsSolver at all to do simulations with flow only in the fractures. Maybe the update of the effective aperture could be called in the flow solver assuming that the aperture is known and that could be either an input when there is no mechanics. Alternatively we could directly ask the user to provide a fracture permeability when there is no mechanics.